python/lsst/ts/m2/tcp_client.py

```diff
@@ -248,7 +248,14 @@ async def _basic_close(self):
         self._monitor_loop_task.cancel()
 
         if self.writer is not None:
-            await tcpip.close_stream_writer(self.writer)
+            # Ignore the TimeoutError (or others) from
+            # asyncio/selector_events.py. A related discussion is here:
+            # https://github.com/home-assistant/core/issues/10468
+            try:
+                await tcpip.close_stream_writer(self.writer)
+            except Exception as error:
+                self.log.debug(f"Error in self._basic_close(): {error}")
```

Comment:
Consider doing the following instead:

```
except Exception:
    self.log.exception("Error closing stream writer. Ignoring.")
```

This way, we will get the fill traceback of the exception.

Reply:
Updated in the commit: [974f955](https://github.com/lsst-ts/ts_m2/pull/27/commits/974f9550581173e28bc58ecf541a3cc7f3a07029). Thanks!